src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2543,8 +2566,11 @@ namespace winrt::TerminalApp::implementation
     // - eventArgs: the event's constituent arguments
     void TerminalPage::_OnTabCloseRequested(const IInspectable& /*sender*/, const MUX::Controls::TabViewTabCloseRequestedEventArgs& eventArgs)
     {
-        const auto tabViewItem = eventArgs.Tab();
-        _RemoveTabViewItem(tabViewItem);
+		const auto tabViewItem = eventArgs.Tab();
```

Comment:
This is probably where your code format woe lies

Reply:
What The F... ormatting :smile: